### PR TITLE
Add tests for `CertificatesOps` `add_certificates_batch` `device`

### LINKED ISSUE #5468: Tests of `add_certificates_batch` for `DeviceCertificate`

### PR DESCRIPTION
**Pull request by TimeEngineer**
closes #5468

### CODE REVIEW

File: libparsec/crates/client/src/certificates_ops/add.rs [hunk 1]
```diff
@@ -102,7 +102,7 @@ pub enum AddCertificateError {
     Internal(#[from] anyhow::Error),
 }
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq, Eq)]
```
Comment (touilleMan) [inline libparsec/crates/client/src/certificates_ops/add.rs:105]: use `p_assert_matches` instead

File: libparsec/crates/testbed/src/env.rs [hunk 2]
```diff
@@ -277,6 +277,130 @@ impl TestbedEnv {
             .unwrap_or_else(|_| panic!("Invalid DeviceID !"));
         self.seal_and(|cache| cache.local_device(self, device_id))
     }
+
```
Comment (touilleMan) [inline libparsec/crates/testbed/src/env.rs:280]: same changes than in #5467  ?

**Review by touilleMan — CHANGES_REQUESTED** [review]

File: libparsec/crates/client/src/certificates_ops/add.rs (hunk 1 above)
Comment (TimeEngineer) [reply libparsec/crates/client/src/certificates_ops/add.rs]: Done

File: libparsec/crates/testbed/src/env.rs (hunk 2 above)
Comment (TimeEngineer) [reply libparsec/crates/testbed/src/env.rs:280]: Done

**Review by touilleMan — APPROVED** [review]

**Pull request merged**